xcodeproj/internal/opts.bzl

```diff
@@ -464,19 +464,20 @@ def _xcode_std_value(std):
 
 # API
 
-def process_opts(*, ctx, target):
+def process_opts(*, ctx, target, build_settings):
```

Comment:
I noticed that the usage later in this PR passes an empty `dict`. Is there a reason that we can't just default `build_settings` to an empty dict?

Reply:
Because it's used as an `inout` parameter. If it's defaulted then the user doesn't get the result back.